.teamcity/settings.kts

```diff
@@ -0,0 +1,76 @@
+import jetbrains.buildServer.configs.kotlin.v2019_2.*
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.script
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.powerShell
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.pullRequests
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.xmlReport
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.XmlReport
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.nuGetPublish
+import jetbrains.buildServer.configs.kotlin.v2019_2.triggers.vcs
+import jetbrains.buildServer.configs.kotlin.v2019_2.vcs.GitVcsRoot
+
+version = "2021.2"
+
+project {
+    buildType(Shimgen)
+}
+
+object Shimgen : BuildType({
+    id = AbsoluteId("Shimgen")
+    name = "Build"
+
+    artifactRules = """
+    """.trimIndent()
+
+    params {
+        password("env.EAZFUSCATOR_NET_LICENSE", "credentialsJSON:cdac4820-7cd7-54e4-b539-8ecaa3490a3c", display = ParameterDisplay.HIDDEN, readOnly = true)
```

Comment:
We're not actually obfuscating this assembly are we?

Reply:
That is a good catch, I will get this removed, if it isn't already in my local changes.